### PR TITLE
GEOPY-1527

### PR DESCRIPTION
**Pull request by domfournier**

### CODE REVIEW

File: curve_apps/edge_detection/driver.py [hunk 1]
```diff
@@ -126,8 +131,25 @@ def get_edges(
...
-        grid.add_data({"canny filter": {"values": edges.flatten(order="F")}})
 
+        return edges
+
+    @staticmethod
+    def get_edges(
+        grid: Grid2D,
+        edges: np.ndarray,
+        detection: DetectionParameters,
+    ) -> tuple:
```
Comment (benk-mira) [inline curve_apps/edge_detection/driver.py:142]:
```suggestion
    ) -> Sequence[np.ndarray]:
```

Comment (benk-mira) [reply curve_apps/edge_detection/driver.py:142]: ..requires `from typing import Sequence`

**Review by benk-mira — CHANGES_REQUESTED** [review]
small typing suggestion

File: curve_apps/edge_detection/driver.py (hunk 1 above)
Comment (domfournier) [reply curve_apps/edge_detection/driver.py]: Used regular tuple[] typing instead.

**Review by benk-mira — APPROVED** [review]

**Pull request merged**